var/spack/repos/builtin/packages/py-typed-ast/package.py

```diff
@@ -12,6 +12,7 @@ class PyTypedAst(PythonPackage):
     homepage = "https://github.com/python/typed_ast"
     pypi = "typed-ast/typed_ast-1.4.0.tar.gz"
 
+    version('1.5.0', sha256='ff4ad88271aa7a55f19b6a161ed44e088c393846d954729549e3cde8257747bb')
     version('1.4.3', sha256='fb1bbeac803adea29cedd70781399c99138358c26d05fcbd23c13016b7f5ec65')
     version('1.4.2', sha256='9fc0b3cb5d1720e7141d103cf4819aea239f7d136acf9ee4a69b047b7986175a')
     version('1.4.1', sha256='8c8aaad94455178e3187ab22c8b01a3837f8ee50e09cf31f1ba129eb293ec30b')
```

Comment:
Can you add:
```python
depends_on('python@3.6:', when='@1.5:', type=('build', 'link', 'run'))
```
and also specify the dep type for the other python dep that's missing it?

Reply:
@dev-zero

Reply:
done, thanks for the reminder